spotpy/unittests/test_database.py

```diff
@@ -108,7 +112,7 @@ def test_sql_multiline_false(self):
         sqldata = sql.getdata()
         self.assertEqual(str(type(sqldata)), str(type(np.array([]))))
         self.assertEqual(len(sqldata[0]), 7)
-        self.assertEqual(len(sqldata), 2)
+        self.assertEqual(len(sqldata), 1)
```

Comment:
This test was incorrect. Since `save` is called only once, ideally, it should have had only one record in the database, but because it used to make an entry even during initialization, it would have 2 records. Issue #106